doc/mkdocs/docs/features/parsing/json_lines.md

```diff
@@ -0,0 +1,49 @@
+# JSON Lines
+
+The [JSON Lines](https://jsonlines.org) format is a text format of newline-delimited JSON. In particular:
+
+1. The input must be UTF-8 encoded.
+2. Every line must be a valid JSON value.
+3. The line separator must be `\n`. As `\r` is silently ignored, `\r\n` is also supported.
+4. The final character may be `\n`, but is not required to be one.
+
+!!! example "JSON Text example"
+
+    ```json
+    {"name": "Gilbert", "wins": [["straight", "7♣"], ["one pair", "10♥"]]}
+    {"name": "Alexa", "wins": [["two pair", "4♠"], ["two pair", "9♠"]]}
+    {"name": "May", "wins": []}
+    {"name": "Deloise", "wins": [["three of a kind", "5♣"]]}
+    ```
+
+JSON Lines input with more than one value is treated as invalid JSON by the [`parse`](../../api/basic_json/parse.md) or
+[`accept`](../../api/basic_json/accept.md) functions. The process it line by line, functions like
```

Comment:
@nlohmann Should `The process it line by line` be `The process is line by line`?

Reply:
Good catch. Fixed in 4b6220acf2b6780350df36df12afc18eeb24a50c.